pkgs/applications/office/ledger/default.nix

```diff
@@ -1,23 +1,24 @@
 { stdenv, lib, fetchFromGitHub, cmake, boost, gmp, mpfr, libedit, python3
-, fetchpatch, installShellFiles, texinfo, gnused, usePython ? true }:
+, installShellFiles, texinfo, gnused, usePython ? false }:
 
 stdenv.mkDerivation rec {
   pname = "ledger";
-  version = "3.2.1";
+  version = "3.3.0";
 
   src = fetchFromGitHub {
     owner  = "ledger";
     repo   = "ledger";
     rev    = "v${version}";
-    sha256 = "0x6jxwss3wwzbzlwmnwb8yzjk8f9wfawif4f1b74z2qg6hc4r7f6";
+    hash   = "sha256-0hN6Hpmgwb3naV2K1fxX0OyH0IyCQAh1nZ9TMNAutic=";
   };
 
-  outputs = [ "out" "dev" "py" ];
+  outputs = [ "out" "dev" ] ++ lib.optionals usePython [ "py" ];
 
   buildInputs = [
-    (boost.override { enablePython = usePython; python = python3; })
     gmp mpfr libedit gnused
-  ] ++ lib.optional usePython python3;
+  ] ++ (if usePython
+        then [ python3 (boost.override { enablePython = usePython; python = python3; })]
+        else [ boost ]);
```

Comment:
What is the motivation for and benefit of this change?

Reply:
I added it to make the default expression don't depend on python. For those who want to use the python bindings, it's still available on `python3Packages.ledger`

Reply:
I see, thanks for the clarification @marsam.